browser.js

```diff
@@ -98,27 +98,32 @@ function changeTab(next) {
 	$(`a[href$="${pages[ret]}"]`).click();
 }
 
+// sets interval to wait for selector to be ready before firing callback
+function waitFor(selectorString) {
+	return new Promise(resolve => {
+		const awaitReact = setInterval(() => {
+			if ($(selectorString)) {
```

Comment:
You're getting the element twice here. Instead get the element, assign to variable, check variable, and then pass that variable to resolve.